# Índice de Pegada de Carbono Alimentar 2018

Dataset disponível no site: https://www.nu3.de/blogs/nutrition/food-carbon-footprint-index-2018

Para aprofundar o debate acerca do indíce de pegada de carbono baseado na produção/consumo de alimentos, foi realizado um estudo que analisa a pegada de carbono da indústria de alimentos, comparando diretamente diferentes dietas em termos de emissões de dióxido de carbono. A pesquisa, que revela as emissões anuais de CO2 por pessoa para 130 nações em todo o mundo, mostra quais países poderiam reduzir significativamente sua pegada de carbono mudando para uma dieta baseada em vegetais, bem como quais tipos de alimentos geram as maiores emissões de dióxido de carbono.

O estudo analisa dados da Organização das Nações Unidas para Agricultura e Alimentação (FAO) para determinar a quantidade de produtos fornecidos para consumo de 11 tipos de alimentos para todos os países pesquisados. Usando dados de emissões de CO2, a pegada de carbono per capita é então calculada para cada tipo de alimento. Isso permite uma comparação direta do CO2 produzido para cada tipo de alimento, bem como entre sete produtos de origem animal e quatro produtos não animais. Por fim, para determinar a classificação final do índice, foi calculada a diferença entre o total de CO2 produzido pela quantidade de produto animal ingerido por uma pessoa em um país versus o total de produto não animal. Este número revela a diferença em kgs de CO2 emitidos por produto animal versus produto não animal. O resultado é um balanço que serve como um indicador do potencial de economia da pegada de carbono por cidadão por ano, se eles mudarem de uma dieta animal para uma dieta não animal.

As perguntas que queremos responder são:

1 - Qual a **quantidade total** de alimentos de origem animal, em kg, que o **Brasil** fornece por pessoa/ano (relativo aos produtos citados na pesquisa)? E de origem vegetal? Qual a **média** e **mediana** das quantidades fornecidas?

2 - Qual a **quantidade total** de CO2, em kg, que o **Brasil** produz por pessoa/ano consumindo produtos de origem animal (relativo aos produtos citados na pesquisa)? E de origem vegetal? Como esse valor é comparado com a média mundial?

3 - Qual a diferença entre o consumo de produtos de origem animal e de origem vegetal no Brasil?

4 - Quais os **5 países** que mais emitem CO2, em kg, por pessoa/ano consumindo produtos de origem animal? E quais países são os que menos consomem?

5 - Em qual posição o **Brasil** se encontra na lista de países que mais consomem produtos de origem animal?

6 - Em qual posição o **Brasil** se encontra na lista de países que mais consomem produtos de origem vegetal?

# Começando nossa pesquisa:

O primeiro passo é importar as biblioteca que nos ajudarão na exploração e visualização dos dados. Assim sendo, iremos abrir o arquivo e visualizar as primeiras 11 linhas para uma rápida análise de nossos dados.

In [298]:
# importando bibliotecas

In [299]:
%matplotlib inline
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns
import numpy as np

In [300]:
# lendo o arquivo

In [301]:
food = pd.read_csv("food_consumption.csv", index_col=[0])

In [302]:
# Primeiras 11 linhas do DataFrame (que foi chamado de variável: food)

In [303]:
print(food.head(11))

      country food_category  consumption  co2_emission
1   Argentina          pork        10.51         37.20
2   Argentina       poultry        38.66         41.53
3   Argentina          beef        55.48       1712.00
4   Argentina     lamb_goat         1.56         54.63
5   Argentina          fish         4.36          6.96
6   Argentina          eggs        11.39         10.46
7   Argentina         dairy       195.08        277.87
8   Argentina         wheat       103.11         19.66
9   Argentina          rice         8.77         11.22
10  Argentina      soybeans         0.00          0.00
11  Argentina          nuts         0.49          0.87


In [304]:
# Quantidade de linhas e colunas do DataFrame

In [305]:
print(food.shape)

(1430, 4)


In [306]:
# Nome das colunas do DataFrame

In [307]:
print(food.columns)

Index(['country', 'food_category', 'consumption', 'co2_emission'], dtype='object')


# Entendendo nossos dados:

Este estudo analisa 130 países ao redor do mundo para determinar a quantidade de dióxido de carbono (CO2) que cada país emite, com base em sua dieta. Isso é calculado através da quantidade de alimentos fornecidos para consumo e quantos kg de CO2 por kg de produto, por pessoa anualmente. Esses dados em nível de país são divididos entre dietas baseadas em animais e não baseadas em animais.

Os dados analisam a quantidade de produtos animais (porco, aves, carne bovina, ovina e caprina, peixe, ovos e leite - incluindo queijo) e produtos não animais (trigo e produtos de trigo, arroz, soja, nozes - incluindo manteiga de amendoim ) que é fornecido para consumo em cada país.

consumption => Fornecido para consumo (kg/pessoa/ano): A quantidade de alimentos, em kg, que um país fornece para consumo. Como se trata de um valor de oferta destinado ao consumo, também pode incluir uma porcentagem de alimentos que não seriam consumidos, como o desperdício de alimentos.

co2_emission => Kg CO2/pessoa/ano: Com base na quantidade de alimentos fornecidos para consumo, este número indica quantos kg de CO2 são produzidos anualmente por pessoa.

# 1 - Qual a quantidade total de alimentos de origem animal, em kg, que o Brasil fornece por pessoa/ano (relativo aos produtos citados na pesquisa)? E de origem vegetal? Qual a média e mediana das quantidades fornecidas?

In [308]:
# Filtrar para Brasil

In [309]:
animal = ['pork', 'poultry', 'beef', 'lamb_goat', 'fish', 'eggs', 'dairy']
veg = ['wheat', 'rice', 'soybeans', 'nuts']
br_animal = food[(food['country'] == 'Brazil') & (food.food_category.isin(animal))]
br_vegetal = food[(food['country'] == 'Brazil') & (food.food_category.isin(veg))]

In [310]:
# Calcular a quantidade total dos alimentos de origem animal listados em kg/pessoa/ano

In [311]:
print(br_animal.groupby("country")['consumption'].agg(['sum', np.mean, np.median]))

            sum       mean  median
country                           
Brazil   265.74  37.962857    12.6


In [312]:
# Calcular a quantidade total dos alimentos de origem vegetal listados em kg/pessoa/ano

In [313]:
print(br_vegetal.groupby("country")['consumption'].agg(['sum', np.mean, np.median]))

           sum    mean  median
country                       
Brazil   89.42  22.355  17.875


In [314]:
# Mostrar DataFrame apenas no Brasil

In [315]:
food_brasil = food[food['country'] == 'Brazil']

In [316]:
print(food_brasil.head(11))

   country food_category  consumption  co2_emission
89  Brazil          pork        12.60         44.60
90  Brazil       poultry        45.00         48.34
91  Brazil          beef        39.25       1211.17
92  Brazil     lamb_goat         0.62         21.71
93  Brazil          fish        10.01         15.98
94  Brazil          eggs         8.98          8.25
95  Brazil         dairy       149.28        212.63
96  Brazil         wheat        53.00         10.11
97  Brazil          rice        32.13         41.12
98  Brazil      soybeans         3.62          1.63
99  Brazil          nuts         0.67          1.19


# 2 - Qual a **quantidade total** de CO2, em kg, que o **Brasil** produz por pessoa/ano consumindo produtos de origem animal (relativo aos produtos citados na pesquisa)? E de origem vegetal? Como esse valor é comparado com a média mundial?


In [317]:
# origem animal:
print("origem animal:")
print(br_animal.groupby("country")['co2_emission'].sum())
print("")
print("origem vegetal:")
# origem vegetal:
print(br_vegetal.groupby("country")['co2_emission'].sum())


origem animal:
country
Brazil    1562.68
Name: co2_emission, dtype: float64

origem vegetal:
country
Brazil    54.05
Name: co2_emission, dtype: float64


In [318]:
# filtrando as categorias de alimentos para todos os produtos de origem animal listados
food_animal = food[(food["food_category"] == 'pork') | 
                   (food["food_category"] == 'poultry') | 
                   (food["food_category"] == 'beef') | 
                   (food["food_category"] == 'lamb_goat') | 
                   (food["food_category"] == 'fish') | 
                   (food["food_category"] == 'eggs') | 
                   (food["food_category"] == 'dairy')
                  ] 
# Encontrando a média a nivel mundial de CO2 produzido pelo consumo de produtos de origem animal
print(food_animal['co2_emission'].sum() / len(food["country"].unique()))

759.2775384615384


Podemos observar que a produção de CO2 no Brasil está maior que a média mundial. Enquanto a média mundial é de 759.28 kg de CO2 por pessoa/ano, o Brasil produz 1562.68 kg. Isto equivale a mais que o dobro da média.

# 3 - Qual a diferença entre o consumo de produtos de origem animal e de origem vegetal no Brasil?

In [319]:
br_animal_co2 = br_animal.groupby("country")['co2_emission'].sum()
br_vegetal_co2 = br_vegetal.groupby("country")['co2_emission'].sum()
br_vegano = br_animal_co2 - br_vegetal_co2
print(br_vegano)

country
Brazil    1508.63
Name: co2_emission, dtype: float64


A diferença é de aproximadamente 1508 kg de CO2. Isso mostra que no Brasil, existe muito mais consumo de produtos de origem animal do que produtos de origem vegetal.

# 4 - Quais os 5 países que mais emitem CO2, em kg, por pessoa/ano consumindo produtos de origem animal? E quais sao os países que menos emitem?

In [320]:
# agrupando e calculando a soma da emissão de CO2 por país
print("países que mais consomem:")
print(food_animal.groupby('country')['co2_emission'].sum().sort_values(ascending=False).head())
print("")
print("países que menos consomem:")
print(food_animal.groupby('country')['co2_emission'].sum().sort_values().head())

países que mais consomem:
country
Argentina      2140.65
Australia      1895.56
Albania        1733.73
New Zealand    1709.81
Iceland        1705.55
Name: co2_emission, dtype: float64

países que menos consomem:
country
Liberia        77.44
Mozambique    104.02
Sri Lanka     145.06
Togo          146.02
Bangladesh    149.55
Name: co2_emission, dtype: float64


Uma informação que podemos analisar através do estudo dos dados é que países da África consomem menos produtos de origem animal. A África é o continente onde hoje se concentram os países mais pobres do mundo. De acordo com uma reportagem da BBC realizada em 9 de fevereiro de 2019, entitulada: **Carne na alimentação: quais países lideram o ranking?**, é relatado: 

"No outro extremo do espectro, em muitos dos países mais pobres do mundo, come-se pouca carne. O etíope médio, por exemplo, consome apenas 7 kg, os ruandeses, 8 kg, e os nigerianos, 9 kg. Trata-se de um patamar dez vezes menor que o da média europeia."

In [321]:
# agrupando e calculando a soma do fornecimento de produtos de origem animal por país
print(food_animal.groupby('country')['consumption'].sum().sort_values().head())

country
Rwanda        17.83
Mozambique    22.71
Liberia       23.40
Malawi        27.64
Zambia        29.62
Name: consumption, dtype: float64


# 5 - Em qual posição o Brasil se encontra na lista de países que mais consomem produtos de origem animal?

In [322]:
print(food_animal.groupby('country')['co2_emission'].sum().sort_values(ascending=False))

country
Argentina      2140.65
Australia      1895.56
Albania        1733.73
New Zealand    1709.81
Iceland        1705.55
                ...   
Bangladesh      149.55
Togo            146.02
Sri Lanka       145.06
Mozambique      104.02
Liberia          77.44
Name: co2_emission, Length: 130, dtype: float64


In [323]:
food_animal_sum = food_animal.groupby('country')['co2_emission'].sum().sort_values(ascending=False)

In [324]:
# transformando em DataFrame para análise e criando a coluna rank
tabela_animal_co2 = pd.DataFrame(food_animal_sum)
tabela_animal_co2["rank"] = tabela_animal_co2["co2_emission"].rank(ascending=False)

In [325]:
print(tabela_animal_co2)

             co2_emission   rank
country                         
Argentina         2140.65    1.0
Australia         1895.56    2.0
Albania           1733.73    3.0
New Zealand       1709.81    4.0
Iceland           1705.55    5.0
...                   ...    ...
Bangladesh         149.55  126.0
Togo               146.02  127.0
Sri Lanka          145.06  128.0
Mozambique         104.02  129.0
Liberia             77.44  130.0

[130 rows x 2 columns]


In [326]:
# encontrando a posição do Brasil nos países que mais produzem CO2 através do consumo de produto animal 
print(tabela_animal_co2.loc["Brazil"])

co2_emission    1562.68
rank               9.00
Name: Brazil, dtype: float64


# 6 - Em qual posição o Brasil se encontra na lista de países que mais consomem produtos de origem vegetal?

In [327]:
food_vegetal = food[(food["food_category"] == 'wheat') | 
                   (food["food_category"] == 'rice') | 
                   (food["food_category"] == 'soybeans') | 
                   (food["food_category"] == 'nuts')
                  ]

food_vegetal_sum = food_vegetal.groupby('country')['co2_emission'].sum().sort_values(ascending=False)

In [328]:
# transformando em DataFrame para análise e criando a coluna rank
tabela_vegetal_co2 = pd.DataFrame(food_vegetal_sum)
tabela_vegetal_co2["rank"] = tabela_vegetal_co2["co2_emission"].rank(ascending=False)

In [329]:
print(tabela_vegetal_co2)

            co2_emission   rank
country                        
Bangladesh        224.63    1.0
Cambodia          208.28    2.0
Vietnam           200.70    3.0
Indonesia         186.33    4.0
Myanmar           179.27    5.0
...                  ...    ...
Namibia            16.27  126.0
Rwanda             15.19  127.0
Paraguay           15.12  128.0
Uganda             14.54  129.0
Ethiopia           11.18  130.0

[130 rows x 2 columns]


In [330]:
# encontrando a posição do Brasil nos países que mais produzem CO2 através do consumo de produto vegetal
print(tabela_vegetal_co2.loc["Brazil"])

co2_emission    54.05
rank            44.00
Name: Brazil, dtype: float64


# Conclusão:

O Brasil é o **nono (9)** país que mais emite CO2 através do consumo de produtos de origem animal e o **quadragésimo quarto (44)** que mais emite CO2 através do consumo de produtos de origem vegetal. Isso nos mostra que o país consome muito mais produtos de origem animal do que de origem vegetal. Nesta pesquisa, foram estudados **130** países. Através do consumo de produtos de origem animal, cada brasileiro produz em média **1.508 kg de CO2**. Para se ter uma ideia, um estudo realizado no Reino Unido mostra que um carro, em média, tem uma emissão de **2.630 kg de CO2** ao ano. 

Através de nossa análise de dados, nos é mostrado a importância de uma transformação no mindset do brasileiro em relação ao consumo exagerado de produtos de origem animal. É importante o começo de um processo de educação no povo brasileiro, para que o mesmo, comece a enxergar e entender a importância do aumento de consumo de produtos vegetais, **tanto para a saúde do corpo, como para a saúde do planeta**.